lib/railsbytes/command.rb

```diff
@@ -25,7 +25,7 @@ def execute(*)
     # @api public
     def prompt(**options)
       require 'tty-prompt'
-      TTY::Prompt.new(options)
+      TTY::Prompt.new(**options)
```

Comment:
Otherwise it fails with:
```
/Users/andyw8/.gem/ruby/3.1.0/gems/tty-prompt-0.23.1/lib/tty/prompt.rb:120:in `initialize': wrong number of arguments (given 1, expected 0) (ArgumentError)
```